.azuredevops/components/rocMLIR.yml

```diff
@@ -0,0 +1,41 @@
+parameters:
+- name: checkoutRepo
+  type: string
+  default: 'self'
+- name: checkoutRef
+  type: string
+  default: ''
+- name: aptPackages
+  type: object
+  default:
+    - cmake
+    - ninja-build
+
+jobs:
+- job: rocMLIR
+  variables:
+  - group: common
+  - template: /.azuredevops/variables-global.yml
+  pool: ${{ variables.HIGH_END_BUILD_POOL }}
```

Comment:
Do we need to change this to LOW_END_BUILD_POOL or cloud?

Reply:
Good catch